diff -aurN darktable-1.0.5.orig/src/iop/watermark.c darktable-1.0.5/src/iop/watermark.c
--- darktable-1.0.5.orig/src/iop/watermark.c	2012-11-09 13:03:37.365726936 +0100
+++ darktable-1.0.5/src/iop/watermark.c	2012-11-09 13:04:31.331672597 +0100
@@ -38,7 +38,9 @@
 #include <inttypes.h>
 
 #include <librsvg/rsvg.h>
+#ifndef RSVG_CAIRO_H
 #include <librsvg/rsvg-cairo.h>
+#endif
 
 #include "common/metadata.h"
 #include "common/utility.h"
