diff -aurN darktable-1.2.2.orig/src/CMakeLists.txt darktable-1.2.2/src/CMakeLists.txt
--- darktable-1.2.2.orig/src/CMakeLists.txt	2013-07-19 19:12:42.267177287 +0200
+++ darktable-1.2.2/src/CMakeLists.txt	2013-07-19 19:13:22.523696467 +0200
@@ -327,23 +327,7 @@
   endif()
 endif(NOT APPLE AND NOT CMAKE_C_COMPILER_ID STREQUAL "Clang")
 
-if(NOT BINARY_PACKAGE_BUILD)
-	MESSAGE("-- Checking for -march=native support")
-	CHECK_C_COMPILER_FLAG("-march=native" MARCHNATIVE)
-	if (MARCHNATIVE)
-		set(MARCH "-march=native")
-	else()
-		MESSAGE("-- Checking for -mtune=native support")
-		CHECK_C_COMPILER_FLAG("-mtune=native" MTUNENATIVE)
-		if (MTUNENATIVE)
-			set(MARCH "-mtune=native")
-		else()
-			set(MARCH "-mtune=generic")
-		endif()
-	endif()
-else()
- 	set(MARCH "-mtune=generic")
-endif()
+set(MARCH "")
 
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -msse2 -g")
 set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -O3 -ffast-math ${MARCH}")
